lib/rspec/rails/view_rendering.rb

```diff
@@ -57,6 +60,44 @@ def find_templates(*args)
             )
           end
         end
+
+        # Delegates all methods to the submitted resolver and for all methods
+        # that return a collection of `ActionView::Template` instances, return
+        # templates with modified source
+        #
+        # @private
+        class ResolverDecorator
+          def initialize(resolver)
+            @resolver = resolver
+          end
+
+          def method_missing(name, *args, &block)
+            result = @resolver.send(name, *args, &block)
+            nullify_templates(result)
```

Comment:
I realize that inspecting return values’ type isn't particularly good design. The alternative is to override `#find_all` and `#find_all_anywhere` depending on Rails version – as far as I recall `#find_all` changed arity from 5 to 6 at some point.